content/components/pages/OverviewPage.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import {
+  useEnvironmentState,
+  useSelectedNormandyEnvironmentAPI,
+} from "devtools/contexts/environment";
+import { PendingReviews } from "../overview/PendingReviews";
+import { Loader } from "rsuite";
+
+export const OverviewPage: React.FC<any> = ({}) => {
+  const { selectedKey: environmentKey } = useEnvironmentState();
+  const normandyApi = useSelectedNormandyEnvironmentAPI();
+
+  const [data, setData] = React.useState([]);
+  React.useEffect(() => {
+    getPendingReviews();
+  }, [environmentKey]);
+
+  const getPendingReviews = async () => {
+    const approvalRequests = await normandyApi.fetchApprovalRequests({
+      approved: "pending",
+    });
+
+    let recipeList = [];
+
+    if (approvalRequests.length) {
+      await Promise.all(
+        approvalRequests.map(async (approvalRequest) => {
+          const {
+            revision: { recipe_id },
+          } = approvalRequest;
+          const recipe = await normandyApi.fetchRecipe(recipe_id);
+          recipeList = [...recipeList, recipe];
```

Comment:
Since everything is local, `recipeList` wasn't passed in as an argument, and we don't need to make this change its identify any more than `let recipeList` already does, this can be the simpler version:

```suggestion
          recipeList.push(recipe);
```